source/franka_matlab/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Franka Matlab changelog
 
+## 0.3.1 (23-03-2023)
```

Comment:
Please update the date, and after we can merge this since the artifact is also published to FW.